webhook/webhook_test.go

```diff
@@ -175,7 +173,7 @@ func TestValidCreateResourceSucceedsWithRoundTripAndDefaultPatch(t *testing.T) {
 	})
 }
 
-func createInnerDefaultResourceWithoutSpec() []byte {
+func createInnerDefaultResourceWithoutSpec(t *testing.T) []byte {
```

Comment:
since you chose to pass `t` inside, then call t.Helper() at the top

Reply:
Done.